tests/collection.js

```diff
@@ -157,7 +150,7 @@ describe( 'Collection', () => {
 
 				collection.add( item );
 
-				expect( item ).to.have.property( 'id', '3' );
+				expect( item.id ).to.be.above( '3' );
```

Comment:
OK, so... hm, since we can't stub `uid()` it looks like the only way to check this is by adding e.g. 100 items (more than could be added above, in previous tests) with ids (like `collection.add( getItem( '0' ) )`) and then adding one without id and checking whether its 100 + 1.